consensus/src/processes/pruning.rs

```diff
@@ -490,6 +490,8 @@ impl<
         pruning_info: PruningPointInfo,
         syncer_sink: Hash,
     ) -> PruningImportResult<VecDeque<Hash>> {
+        // Function returns the pruning points on the path
```

Comment:
It's better to put this above function declaration. Generally we put comments above the declaration if they describe API or function constraints, and inside the body if they detail implementation details.